magda-web-client/src/Components/Dataset/Add/Pages/DetailsAndContents/index.tsx

```diff
@@ -0,0 +1,350 @@
+                    <h4>Which themes does this dataset cover?</h4>
+                    <ToolTip>
+                        Themes are the topics your dataset covers and they help
+                        people find related datasets within a topic. We
+                        recommend themes are kept to 5-10 topics. We've
+                        identified themes from your document, that are
+                        consistent with similar datasets.
+                    </ToolTip>
+                    <div className="clearfix">
+                        <TagInput
+                            value={dataset.themes}
+                            onChange={editDataset("themes")}
+                            placeHolderText="Enter a theme"
+                            useVocabularyAutoCompleteInput={false}
+                        />
+                    </div>
+                </div>
+
+                <div className="question-description">
+                    <h4>Please add a description for this dataset</h4>
+                    <ToolTip>
+                        A good dataset description clearly and succintly
+                        explains the contents, purpose and value of the dataset.
+                        This is how users primarily identify and select your
+                        dataset from others. Here you can also include
+                        information that you have not already covered in the
+                        metadata.
+                    </ToolTip>
+                    <div className="clearfix">
+                        <MultilineTextEditor
+                            value={dataset.description}
+                            placerHolder="Enter description text"
+                            limit={250}
+                            onChange={props.edit("dataset")("description")}
+                        />
+                    </div>
+                </div>
+
+                <h3 className="with-underline">Dates and updates</h3>
+
+                <div className="question-dataset-status">
+                    <h4>What is the status of the this dataset?</h4>
+                    <div className="row">
+                        <div className="col-sm-6">
+                            <ReactSelect
+                                className="react-select"
+                                isMulti={false}
+                                isSearchable={false}
+                                options={Object.keys(
+                                    codelists.datasetCurrencyStatus
+                                ).map(key => ({
+                                    label: codelists.datasetCurrencyStatus[key],
+                                    value: key
+                                }))}
+                                onChange={(item: any) => {
+                                    const status = item.value as CurrentStatusType;
+                                    editCurrency("status")(status);
```

Comment:
It'd be a bit nicer for the user if we didn't reset values when the status option changed. With the way if it is now, if I select "superseded" and put in a dataset, then select "retired" to see what that does, realised I actually want "superseded" and change it back, I've got to put in the dataset again.

Instead we could just keep that data, and do logic on submit - e.g. if status !== "SUPERSEDED" then don't set `supersededBy`.

Reply:
Sounds a good idea~ Will update accordingly~